sys/shell/commands/sc_netif.c

```diff
@@ -143,6 +144,10 @@ static void _set_usage(char *cmd_name)
          "       * \"chan\" - alias for \"channel\"\n"
          "       * \"csma_retries\" - set max. number of channel access attempts\n"
          "       * \"cca_threshold\" - set ED threshold during CCA in dBm\n"
+#ifdef MODULE_L2FILTER
+         "       * \"l2filter\" - add link layer address to filter\n"
+         "       * \"l2filter_rm\" - remove link layer address from filter\n"
```

Comment:
Can the command be `ifconfig l2filter add` or `ifconfig l2filter del` please? IP addresses are also lists of addresses and are configured that way.